app/controllers/import_controller.rb

```diff
@@ -382,6 +386,34 @@ def validate_workflow_specific_enums(workflow, field, value, row_ind)
     end
     raise ValidationError.new(err_msg, row_ind)
   end
+
+  def validate_continuous_exposure(workflow, field, continuous_exposure_value, row_ind, last_date_of_exposure_value)
+    return false if workflow == :isolation || continuous_exposure_value.blank?
+
+    continuous_exposure_boolean = import_and_validate_bool_field(field, continuous_exposure_value, row_ind)
+
+    if continuous_exposure_boolean
+      if last_date_of_exposure_value.present?
+        err_msg = "Value '#{continuous_exposure_value}' is not valid for '#{VALIDATION[field][:label]}' with " \
+                "'#{VALIDATION[:last_date_of_exposure][:label]}' of '#{last_date_of_exposure_value}.' " \
+                'Monitorees may be imported either with a Last Date of Exposure value or Continuous Exposure ' \
+                'set to \'true.\''
+
+        raise ValidationError.new(err_msg, row_ind)
+      else
+        @warnings[VALIDATION[field][:label]] = 'Your import contains one or more monitorees with Continuous Exposure ' \
+                                               'enabled. Please note that monitorees with Continuous Exposure enabled ' \
+                                               'will receive symptom assessments indefinitely unless a Sara Alert user ' \
+                                               'manually deactivates Continuous Exposure after import. To proceed with ' \
+                                               'this import, select \'Proceed with Import\' to acknowledge you understand ' \
```

Comment:
```suggestion
                                               'this import, select \'Continue\' to acknowledge you understand ' \
```

Reply:
Thank you for catching that! See commit [6e3fbb1](6e3fbb1)